feature-group-definitions/storage-buckets.dist.yml

```diff
@@ -0,0 +1,28 @@
+# Generated from: storage-buckets.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Storage buckets
+spec: https://wicg.github.io/storage-buckets/
+status:
+  baseline: false
+  support:
+    chrome: "122"
```

Comment:
This is correct:
https://chromestatus.com/feature/5739224579964928
https://chromiumdash.appspot.com/commit/834e1568e27cb1f088627f3c9f6c36c2803115f0